bootstrap.sh

```diff
@@ -15,4 +15,4 @@ docker run \
 -e INPUT_APP_ID="$INPUT_APP_ID" \
 -e INPUT_TEST_EVENT_PATH="${TEST_EVENT_PATH}" \
 -e GITHUB_REPOSITORY="$GITHUB_REPOSITORY" \
-hamelsmu/chatops
\ No newline at end of file
+hamelsmu/chatops
```

Comment:
Sorry, the GitHub UI did this. It's just one byte, and it's better form to include a newline at EOF.

If requested, I can switch to using a CLI to suppress this change. (It's fun using the web UI to construct these commits -- try it yourself.)